### PR TITLE
adjust collectedBy uris

### LINKED ISSUE #180: Update URIs for NWBib, Edoweb, RPB & add collectedBy links for ZDB
We alreay link to http://lobid.org/resources/NWBib and http://lobid.org/resources/Edoweb in the data but they currently don't resolve. The RDF descriptions (in Turtle) are lying in an Etherpad of which I won't post the URL here.

### PR DESCRIPTION
**Pull request by acka47**
Fixes #180

### CODE REVIEW

File: src/main/resources/morph-hbz01-to-lobid.xml [hunk 1]
```diff
@@ -1796,8 +1799,8 @@
 		<!-- ########## classification -->
 		<!-- ####################### -->
 		<!--nwbib -->
-		<data source="078n[-1].a" name="http://purl.org/ontology/holding#collectedBy">
-			<regexp match=".*" format="$[ns-lobid-resource]NWBib"/>
+		<data source="078n[-1].a" name="http://purl.org/lobid/lv#inCollection">
+			<regexp match=".*" format="http://lobid.org/resources/HT014176012#!"/>
 		</data>
```
Comment (dr0i) [inline src/main/resources/morph-hbz01-to-lobid.xml]: not wrong, but why not stick to the variable?

File: src/test/resources/hbz01.es.json [hunk 2]
```diff
@@ -83,6 +79,10 @@
   } ],
   "hbzId" : "HT018454638",
   "id" : "http://lobid.org/resources/HT018454638#!",
+  "inCollection" : [ {
+    "id" : "http://lobid.org/resources/HT014176012#!",
+    "label" : "lobid Ressource"
+  } ],
```
Comment (dr0i) [inline src/test/resources/hbz01.es.json]: You loose a nice label here. If you want to keep it, the sidecar approach will not work out-of-the-box. Rather you will have to add it in the morph by setting a new rdf:subject.

**Comment by acka47** [comment]
The build is a succes on my local machine. I don't know what to do.

**Comment by dr0i** [comment]
@acka47 Hm:
1. I started a review by commenting something to the morph. But it was tagged as "pending" and so it could be that only I was able to see this? I explicitly noted now my comments into the review
2. re "building locally": did you "git pull orgin master" in your master branch?

File: src/test/resources/hbz01.es.json (hunk 2 above)
Comment (dr0i) [inline src/test/resources/hbz01.es.json]: Uh - sidecar should(!) work . Just define the URI `http://lobid.org/resources/HT014176012#!` and give it the proper label.

**Comment by acka47** [comment]
I made the requested changes. Although the built still worked on my local machine, it won't work here...

File: src/test/resources/jsonld/BT000040377 [hunk 3]
```diff
@@ -31,6 +27,10 @@
   },
   "hbzId" : "BT000040377",
   "id" : "http://lobid.org/resources/BT000040377#!",
+  "inCollection" : [ {
+    "id" : "http://lobid.org/resources/HT014176012#!",
+    "label" : "Edoweb Rheinland-Pfalz"
+  } ],
   "isPartOf" : [ {
```
Comment (dr0i) [inline src/test/resources/jsonld/BT000040377]: Hm. This should be NWbib put is now RPB.

**Comment by dr0i** [comment]
@fsteeg since the field and the identifier for recognizing the collection has changed please have a look if/how this affects the nwbib-frontend.

**Comment by dr0i** [comment]
Deployed small test , see e.g. http://test.lobid.org/resources/BT000003404?format=json.

**Comment by fsteeg** [comment]
> @fsteeg since the field and the identifier for recognizing the collection has changed please have a look if/how this affects the nwbib-frontend. Have a look at the web test in the commits, too!

NWBib uses API 1.x, so no updates needed here at this point.

**Comment by dr0i** [comment]
Deployed to production, closing.

**Pull request merged**